src/styles/components/_not-found.scss

```diff
@@ -0,0 +1,19 @@
+.q-not-found {
+  .q-icon {
```

Comment:
q-icon class refers to Icon component itself. It would be better if the name of this class is changed to &__icon